README.md

```diff
@@ -1,3 +1,3 @@
-# netuitive.packages.cassandra 1.1.0
+# CloudWisdom.packages.cassandra 1.1.0
```

Comment:
Keep this as netuitive, it's part of a fully qualified name.

Reply:
changed back to netuitive